language_common.md

```diff
@@ -0,0 +1,6 @@
+# Standards common to coding in any language
+
+## Avoid using 'magic' numbers
```

Comment:
Could we expand this to make the point that it is often useful to use constants or (better) enums for booleans too? For example, fileUtility.delete("/home/jblogs/tmp", true) vs fileUtility.delete("/home/jblogs/tmp", RECURSIVE).

Reply:
I think this may be conflating the general advisability of using constants with the specific guidance on magic numbers. How about if I create a separate section on the use of constants?